### PR TITLE
Rewrite/Reboot of /contributors/ page

### PR DESCRIPTION
**Pull request by rbowen**
Complete rewrite of /contributors/ page, with a focus on *only* that audience. Several fixes/tweaks/updates that came along for the ride.

### CODE REVIEW

**Comment by bdelacretaz** [comment]
Hi Rich, would it be possible & convenient for you to work in `preview/*` branches in this repository instead of [your own](https://github.com/rbowen/comdev-site/tree/contrib)? That would enable the preview feature, allowing us to better see the proposed changes.

File: source/contributors/_index.md [hunk 1]
```diff
@@ -4,100 +4,181 @@ url: /contributors/
...
+the [contributor ladder](/contributor-ladder.html).
 
-## Project Independence And Policies
+User &rarr; **Contributor** &rarr; [Committer](/committers/) &rarr; [PMC member](/pmc/)
 
-While not all ASF projects practice all aspects of the Apache Way in the same way, there are a number of rules that ASF 
-projects must follow – things like complying with PMC 
-[release voting][2], [legal policy][3], [brand policy][4], 
-using [mailing lists][5], etc., which are [documented in various places][6]. 
+While we tend to think of contributions to open source as *code*, a
```
Comment (bdelacretaz) [inline source/contributors/_index.md:12]: I would say "while we often think..."

File: source/contributors/_index.md [hunk 2]
```diff
@@ -4,100 +4,181 @@ url: /contributors/
...
+`users-unsubscribe@PROJECT.apache.org` from the email address with which
+you subscribed.
+
+In each case, you'll receive a confirmation email, with further
+instructions.
+
+#### Other communication channels
+
+Some projects use other communication channels, such as Slack, Discord,
+or WiChat. This will (hopefully) be listed on the project page.
```
Comment (bdelacretaz) [inline source/contributors/_index.md:90]: I think that's "WeChat"

File: source/contributors/_index.md [hunk 3]
```diff
@@ -4,100 +4,181 @@ url: /contributors/
...
+project and its specific contributor flow.
 
-There is nothing at the ASF that says you must write code 
-in order to be a committer. Anyone who is supportive by contributing towards project,
-community, documentation or code is a likely candidate for committership.
+We recommend that you start by finding, and joining, the relevant
+communication channel. Introduce yourself, and say what you're
+interested in working on. How people respond to this can be a good
+indicator of how welcoming the project is. It will also be an
+opportunity for them to point you to good first issues,
```
Comment (bdelacretaz) [inline source/contributors/_index.md:101]: we have https://community.apache.org/committers/good-first-issues.html to explain that

File: source/contributors/_index.md [hunk 4]
```diff
@@ -4,100 +4,181 @@ url: /contributors/
...
+
+### Communication
+
+Most (not all) Apache project have their main discussions on [mailing
+lists][lists]. Most projects have
+`users@project.apache.org` and `dev@project.apache.org` mailing lists,
+focused, respectively, on end-user questions, and development activity.
+You can read the archives (i.e., messages from the past), as well as
+participate in current discussions, either by subscribing to the lists,
+or by using the [lists.apache.org][lists] site
```
[the remark this answers is not in the document]
Comment (rbowen) [reply source/contributors/_index.md:65]: It worked in my local test. I'll convert it to a number if you feel that's necessary, but I find this format a lot easier to read.

File: source/contributors/_index.md (hunk 1 above)
Comment (rbowen) [reply source/contributors/_index.md]: Thanks. Fixed.

File: source/contributors/_index.md (hunk 3 above)
Comment (rbowen) [reply source/contributors/_index.md]: Ooh. Right. Thanks.

**Comment by rbowen** [comment]
I am clearly doing something wrong with the preview/ style branch, but will try again next time.

**Comment by rbowen** [comment]
By the way, attempting to move this to a preview branch.

**Comment by rbowen** [comment]
I've pushed a preview to https://community-contrib-01.staged.apache.org/ (I think) and am waiting for that to propagate.

**Comment by bdelacretaz** [comment]
Preview at https://community-contrib-01.staged.apache.org/contributors/ worked indeed, thank you!

+1 to merging, maybe just add the "newcomers" tag so that it appears at https://community.apache.org/tags/newcomers.html

(and we probably need to cleanup the tags at some point, or ongoing, for more consistency)

**Pull request merged**